packages/libcrypto/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "libcrypto"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+releases-url = "https://github.com/aws/aws-lc/commits/fips-2024-09-27/"
```

Comment:
I wonder if we will be able to vend the one version of AWS-LC that all Bottlerocket components consume, with this now we have 3 (aws-lc-rs, libcrypto in the SDK for `gofips`). Doesn't harm but it might be confusing.